### PR TITLE
Adhere to JSON API spec

### PR DESCRIPTION
**Pull request by pjweisberg**
This is to fix the merge conflicts of #115 so that @chb0github can merge it.

### CODE REVIEW

**Comment by pjweisberg** [comment]
The continuous-integration check fails because it can't download some SonarQube library, but a local `mvn clean install` succeeded.

File: katharsis-core/src/main/java/io/katharsis/queryParams/DefaultQueryParamsParser.java [hunk 1]
```diff
@@ -17,65 +17,377 @@
...
+
+            String resourceType = propertyList.get(0);
+
+            if (temporaryGroupingMap.containsKey(resourceType)) {
+                Set<String> resourceParams = temporaryGroupingMap.get(resourceType);
+                resourceParams.addAll(entry.getValue());
+                temporaryGroupingMap.put(resourceType, resourceParams);
+            } else {
+                Set<String> resourceParams = new LinkedHashSet<>();
+                resourceParams.addAll(entry.getValue());
```
Comment (chb0github) [inline katharsis-core/src/main/java/io/katharsis/queryParams/DefaultQueryParamsParser.java]: these 2 lines are redundant and in both clauses. They could be removed.

**Comment by chb0github** [comment]
@meshuga or @masterspambot do either of you know about the problem @pjweisberg is talking about?

Can we get a release, like, tomorrow?

**Comment by pjweisberg** [comment]
Ick, could we _not_ do squash-and-merge with commits from multiple authors please?  Now my name is on that entire change, despite the fact that @dustinstanley wrote almost all of it.

**Comment by chb0github** [comment]
Sorry. I didn't realize it would would do that. If it matters that much I can consider resetting the last change and re-merging the PR if it's possible.

**Pull request merged**